openml-lightgbm/lightgbm-provider/pom.xml

```diff
@@ -125,7 +130,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <environmentVariables>
-                        <LC_ALL>C</LC_ALL>
+                        <!--<LC_ALL>C</LC_ALL>-->
```

Comment:
That was what we fixed :+1: :)

Until now we needed to run LightGBM only with the "C" locale, or scores and tests would break. You can test that by switching back to v3.0.0 in the pom.xml. You'll get a bunch of test errors. It only works when that is set.

The new patch works with any locale.

Reply:
In this case, IMO you should comment the whole configuration tag. Because there is no configuration besides setting LC_ALL environment variable.

Reply:
I  hadn't seen Paulo's comment, I agree with Paulo, we should remove the configuration if we are not using any